javascript/javascript1/week2/index2.js

```diff
@@ -0,0 +1,81 @@
+// #1 Flight booking fullname function.
+
+function getFullname(firstname, surname) {
+  return firstname + " " + surname;
+}
+const fullname1 = getFullname("Emilia", "Tambe");
+const fullname2 = getFullname("Ako", "Ayukngat");
+const fullname = getFullname("Benjamin", " Hughes");
+console.log(fullname1);
+console.log(fullname2);
+console.log(fullname);
+let formalname = "Lord" + " " + fullname;
+if ((useFormalname = true)) {
```

Comment:
`useFormalName` is not declared anywhere. Also `= true` is an assignment, `=== true` is a comparison